Doc/conf.py

```diff
@@ -23,6 +23,10 @@
 except ImportError:
     _tkinter = None
 '''
+
+# Manpage config
```

Comment:
Nit: This comment can be dropped. The name of the setting is already self-descriptive.

Reply:
Done